addons/im_livechat/static/src/embed/core/thread_model_patch.js

```diff
@@ -56,6 +56,10 @@ patch(Thread.prototype, {
         return this.type === "livechat" || super.isChatChannel;
     },
 
+    get typesAllowingCalls() {
+        return super.typesAllowingCalls.concat(["livechat"]);
+    },
```

Comment:
This patch is both in core and embed? 
If that's needed there is a problem with bundles (to fix later), as core should normally be included in embed

Reply:
Yes indeed, the way we split bundles changed a lot since the live chat refactoring. We'll update it to match new bundling later in order to delete those redundancies.